minerl/data/pipeline/render.py

```diff
@@ -245,8 +251,10 @@ def extract(fname):
                 assert not os.stat(action_mcbr).st_size == 0
 
                 # Run the actual parse action and make sure that its actually of length 0.
-                p = subprocess.Popen(["python3", "parse_action.py", os.path.abspath(
-                    action_mcbr)], cwd='action_rendering')
+                p = subprocess.Popen(
+                    ["python3", "parse_action.py", os.path.abspath(action_mcbr)],
+                    cwd='action_rendering',
+                )
```

Comment:
Is this a linting only change? Also is there a way to open the python that is running instead of python3? This seems like it could break things if you had a funny python config

Reply:
yeah, this is linting only.

Reply:
Could try https://stackoverflow.com/a/2589722/1091722 to get path of same python?

Reply:
Yeah something like `subprocess.Popen([sys.executable, "path/to/script.py"])` looks like it would work

Reply:
Tested this out in the latest commit. It looks fine!

Reply:
Awesome thanks!